src/homework/04_iteration/dna.cpp

```diff
@@ -48,27 +48,26 @@ std::string get_dna_complement(string dna)
 {
     string reverseDNa = get_reverse_string(dna);
 
-    for (std::size_t count = 0; count < dna.size(); count ++ )
+    for (std::size_t count = 0; count < dna.size(); count++ )
     {
-        if (reverseDNa[count] == 'A')
-        {
+        if (reverseDNa[count] == 'A'){
+        
             reverseDNa[count] == 'T';
         }
     
-        else if (reverseDNa[count] == 'T')
-        {
+        else if (reverseDNa[count] == 'T'){
+        
             reverseDNa[count] == 'A';
         }
 
-        else if (reverseDNa[count] == 'C')
-        {
+        else if (reverseDNa[count] == 'C'){
+        
             reverseDNa[count] == 'G';
         }
 
         else {reverseDNa[count] == 'C';
         }
-
-    return reverseDNa;    
+        return reverseDNa;    
```

Comment:
Should be:

for (std::size_t count = 0; count < dna.size(); count++ )
    {
        if (reverseDNa[count] == 'A'){
        
            reverseDNa[count] = 'T';
        }
    
        else if (reverseDNa[count] == 'T'){
        
            reverseDNa[count] = 'A';
        }

        else if (reverseDNa[count] == 'C'){
        
            reverseDNa[count] = 'G';
        }

        else {reverseDNa[count] = 'C';
        }
        
    
    }
    
    return reverseDNa;